docs/dev/libvirt/README.md

```diff
@@ -286,6 +286,38 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### Console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console depends on the route which is created by auth operator.
+To make it works we need to first create the manifests and edit the `domain` for ingress config, before directly creating the cluster.
```

Comment:
nit: "works" -> "work".  Alternatively, drop the leader and compact the sentence to "We need to edit the ingress `domain` before creeating the cluster." or some such.